server/app.js

```diff
@@ -13,7 +13,7 @@ app.use(bodyParser.json())
 
 
 //routes
-app.use("/", require("./routes/notes"))
+app.use("/", require("./routes/index"))
```

Comment:
not required index in the end

Reply:
will it look index.js defaulty

Reply:
yes it will. just mention `require('./routes')`

Reply:
ok it's working i changed